From 98fdf18834809543d22b04d626474e30dfcb6236 Mon Sep 17 00:00:00 2001
From: Sam Nazarko <email@samnazarko.co.uk>
Date: Fri, 10 Jun 2016 15:41:35 +0100
Subject: [PATCH] Allow CC and CXX to be passed to ffmpeg autobuild. This will
 allow OSMC build system to pass ccache path for a much quicker build.

Signed-off-by: Sam Nazarko <email@samnazarko.co.uk>
---
 tools/depends/target/ffmpeg/autobuild.sh | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tools/depends/target/ffmpeg/autobuild.sh b/tools/depends/target/ffmpeg/autobuild.sh
index b9bfd57..dfaf016 100755
--- a/tools/depends/target/ffmpeg/autobuild.sh
+++ b/tools/depends/target/ffmpeg/autobuild.sh
@@ -160,6 +160,8 @@ CFLAGS="$CFLAGS" CXXFLAGS="$CXXFLAGS" LDFLAGS="$LDFLAGS" \
 	--disable-mipsdspr1 \
 	--disable-mipsdspr2 \
 	--enable-libdcadec \
+	--cc="$CC"
+	--cxx="$CXX"
         ${FLAGS}
 
 make -j ${BUILDTHREADS} 
-- 
2.7.4

